### PR TITLE
Breaking changes in tslint version 5.  Renamed failureCount to errorCount

### PR DESCRIPTION
**Pull request by markm11**

### CODE REVIEW

**Comment by panuhorsmalahti** [comment]
You should edit index.ts and then compile index.js.

**Comment by markm11** [comment]
OK I just did that.  Sorry about that.

**Comment by Connormiha** [comment]
> You should edit index.ts and then compile index.js.

And also update `package.json` deps

**Comment by chrismbarr** [comment]
Keep in mind that with TSlint 5 rules can now have either **error** or **warning** severity.  You'll need to make sure that both `errorCount` list and `warningCount` are handled.

TSLint 5.0.0 release notes mentions all of this: https://github.com/palantir/tslint/releases/tag/5.0.0

**Comment by panuhorsmalahti** [comment]
Good point, that must be fixed too. There's quite a few changes in tslint 5.0, I'll need to have a better look if other changes are necessary.

File: index.ts [hunk 1]
```diff
@@ -200,7 +200,9 @@ tslintPlugin.report = function(options?: ReportOptions) {
     const reportFailures = function(file: TslintFile) {
 
         if (file.tslint) {
-            const failureCount = file.tslint.failureCount;
+            // Version 5.0.0 of tslint no longer has a failureCount member
+            // It was renamed to errorCount. See tslint issue #2439
+            const failureCount = file.tslint.failureCount || file.tslint.errorCount;
```
Comment (markm11) [inline index.ts]: I remove it.

**Comment by amirburbea** [comment]
So when will this be the version in npm??

**Comment by panuhorsmalahti** [comment]
Released now.

**Pull request merged**